Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -241,6 +246,7 @@ def check_analysis_status_and_get_results_command(intezer_api: IntezerApi, args:
                     continue
                 else:
                     analysis_result = analysis.result()
+                    file_metadata = analysis.get_root_analysis().metadata
```

Comment:
Shouldn't the metadata() function be called?
```suggestion
                    file_metadata = analysis.get_root_analysis().metadata()
```

Reply:
same